packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -163,38 +163,38 @@ export function getComputedFluidTypographyValue( {
 	} );
 
 	// Viewport widths defined for fluid typography. Normalize units
-	const maximumViewPortWidthParsed = getTypographyValueAndUnit(
-		maximumViewPortWidth,
+	const maximumViewportWidthParsed = getTypographyValueAndUnit(
+		maximumViewportWidth,
 		{ coerceTo: fontSizeUnit }
 	);
-	const minumumViewPortWidthParsed = getTypographyValueAndUnit(
-		minimumViewPortWidth,
+	const minumumViewportWidthParsed = getTypographyValueAndUnit(
+		minimumViewportWidth,
 		{ coerceTo: fontSizeUnit }
 	);
 
 	// Protect against unsupported units.
 	if (
-		! maximumViewPortWidthParsed ||
-		! minumumViewPortWidthParsed ||
+		! maximumViewportWidthParsed ||
+		! minumumViewportWidthParsed ||
 		! minimumFontSizeRem
 	) {
 		return null;
 	}
 
 	// Build CSS rule.
 	// Borrowed from https://websemantics.uk/tools/responsive-font-calculator/.
-	const minViewPortWidthOffsetValue = roundToPrecision(
-		minumumViewPortWidthParsed.value / 100,
+	const minViewportWidthOffsetValue = roundToPrecision(
+		minumumViewportWidthParsed.value / 100,
 		3
 	);
 
 	const viewPortWidthOffset =
```

Comment:
This isn't part of the API, but while we're here, shall we change this one to `viewport`, too? And where the const gets used here, of course: https://github.com/WordPress/gutenberg/blob/cacb970df2b3bbc67072f84bdd7ede32612f0451/packages/block-editor/src/components/font-sizes/fluid-utils.js#L202

Reply:
Oh yes! Thank you! 🙇🏻